docs/extensions/extensions.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 8
```

Comment:
Maybe we move this section below "Disabling lint error" seems a bit in middle of how to use SE-2 docs. 

Also I think in [Quickstart/installation#setup](https://docs.scaffoldeth.io/quick-start/installation#setup) we should move  `npx create-eth` option first and clone option second. 

and while mentioning about `npx create-eth` we could add a line that you could use extensions too using `-e` flag and point to this section. Just to want to highlight the extensions at the beginning so people can make decision according to their need